parsers/US_PJM.py

```diff
@@ -106,8 +106,6 @@ def fetch_consumption_forecast_7_days(
 
     if target_datetime:
         raise NotImplementedError("This parser is not yet able to parse past dates")
-    if not session:
-        session = Session()
```

Comment:
This should not be removed, if anythin it's the default value that should be changed to None as using mutable defaults in python can potentially cause all kinds of issues.

Reply:
Thanks, fixed for the whole file!